src/test/ui/repr/repr-transparent-issue-87496.rs

```diff
@@ -0,0 +1,11 @@
+// Regression test for the ICE described in #87496.
+
+// check-pass
```

Comment:
Hmm, for some reason there doesn't seem to be a `.stderr` file being generated? Yet CI is passing...